src/Shared/Components/FeatureDescription/types.ts

```diff
@@ -29,8 +30,12 @@ export interface FeatureDescriptionModalProps {
 }
 
 export interface DescriptorProps extends FeatureDescriptionModalProps {
-    breadCrumbs: Breadcrumb[]
+    breadCrumbs?: Breadcrumb[]
     additionalContainerClasses?: string
     iconClassName?: string
     children?: React.ReactNode
+    showInfoIconTippy?: boolean
+    docLinkText?: string
+    dataTestId?: string
+    additionalContent?: ReactNode
```

Comment:
since these values are only rendered when `showInfoIconTippy` is true 

we can make them conditional

Reply:
As discussed we can use additionalContent props in other places as well in the component so moving with this change